api/chaos/serverexperimentation/v1/serverexperimentation.proto

```diff
@@ -72,6 +72,27 @@ message HTTPFaultConfig {
   }
 }
 
+// The configuration of a Redis fault.
+message RedisFaultConfig {
+  // The targeting of the fault describing what requests are being considered for faults.
+  FaultTargeting fault_targeting = 1 [ (validate.rules).message.required = true ];
+
+  oneof fault {
+    option (validate.required) = true;
+
+    // The error fault responds to redis commands with an error.
+    ErrorFault error_fault = 2;
+    // The latency fault delays redis commands.
+    LatencyFault latency_fault = 3;
+  }
+}
+
+// The definition of a redis error fault.
+message ErrorFault {
```

Comment:
I don't think `AbortFault` maps well to redis since abort codes are not relevant. Let's create a new fault definition here.